src/Components/HeaderCategory.jsx

```diff
@@ -1,7 +1,21 @@
 import React, { Component } from 'react';
-import { Menu } from 'semantic-ui-react'
+import { Menu, Divider, Container } from 'semantic-ui-react'
 
-const categories = ['News', 'Arts', 'Books', 'Business', 'Food', 'Opinion', 'Politics', 'Real Estate', 'Science', 'Sports', 'Style', 'Tech', 'Travel']
+const categories = [
+  {name: 'News', color: 'red'},
+  {name: 'Business', color: 'blue'},
+  {name: 'Tech', color: 'pink'},
+  {name: 'Sports', color: 'teal'},
+  {name: 'Politics', color: 'grey'},
```

Comment:
why dont you fetch the categories from the backend?

Reply:
we are implementing that in another user story (soon to be pr). Can this be merged as is so we avoid unnecessary double work?